release-notes/10.0/preview/preview7/api-diff/Microsoft.NETCore.App/10.0-preview7_System.Runtime.md

```diff
@@ -0,0 +1,42 @@
+      {
++         public static System.UInt128 BigMul(System.UInt128 left, System.UInt128 right, out System.UInt128 lower);
+      }
+      public readonly struct UIntPtr : System.IComparable, System.IComparable<nuint>, System.IEquatable<nuint>, System.IFormattable, System.IParsable<nuint>, System.ISpanFormattable, System.ISpanParsable<nuint>, System.IUtf8SpanFormattable, System.IUtf8SpanParsable<nuint>, System.Numerics.IAdditionOperators<nuint, nuint, nuint>, System.Numerics.IAdditiveIdentity<nuint, nuint>, System.Numerics.IBinaryInteger<nuint>, System.Numerics.IBinaryNumber<nuint>, System.Numerics.IBitwiseOperators<nuint, nuint, nuint>, System.Numerics.IComparisonOperators<nuint, nuint, bool>, System.Numerics.IEqualityOperators<nuint, nuint, bool>, System.Numerics.IDecrementOperators<nuint>, System.Numerics.IDivisionOperators<nuint, nuint, nuint>, System.Numerics.IIncrementOperators<nuint>, System.Numerics.IModulusOperators<nuint, nuint, nuint>, System.Numerics.IMultiplicativeIdentity<nuint, nuint>, System.Numerics.IMultiplyOperators<nuint, nuint, nuint>, System.Numerics.INumber<nuint>, System.Numerics.INumberBase<nuint>, System.Numerics.ISubtractionOperators<nuint, nuint, nuint>, System.Numerics.IUnaryNegationOperators<nuint, nuint>, System.Numerics.IUnaryPlusOperators<nuint, nuint>, System.Numerics.IShiftOperators<nuint, int, nuint>, System.Numerics.IMinMaxValue<nuint>, System.Numerics.IUnsignedNumber<nuint>, System.Runtime.Serialization.ISerializable
+      {
++         public static nuint BigMul(nuint left, nuint right, out nuint lower);
+      }
+  }
+  namespace System.Diagnostics.CodeAnalysis
+  {
++     [System.Runtime.CompilerServices.CompilerLoweringPreserveAttribute]
+      public sealed class DynamicallyAccessedMembersAttribute : System.Attribute
```

Comment:
Somehow I am not seeing this in the code

Reply:
This one. https://github.com/dotnet/runtime/blob/78abe1053d1a1d2134a11254eb8e0216148b528e/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicallyAccessedMembersAttribute.cs#L30

Reply:
https://github.com/dotnet/runtime/commit/757fb9de52ba25b8f8f364f90adc65cc29c27c5d

Reply:
I discovered I didn't pull the latest from the main 😞